### DISCUSSION TITLE
Where could I locate the web browser settings?

### DISCUSSION THREAD

**Discussion by MuerteRoja** [goastian/midori-desktop#60] [Q&A] [answered]
Hello, good morning, I apologize for the inconvenience, I noticed that the Astian search engine is now available to the public and I was wondering where I could find the browser settings, I would be interested in configuring it to my liking
![Buscador web](https://github.com/goastian/midori-desktop/assets/126993006/302e884d-d88a-4f93-b8a8-6786f5f783c1)

**Comment by Ponchale** [comment, marked as answer]
We are relaunching and improving the search engine, that is why we have announced it in the alpha phase, it still needs to be ready but thank you for your interest and enthusiasm.
